SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -229,6 +230,25 @@ void USpatialGameInstance::Init()
 	}
 }
 
+#if ENGINE_MINOR_VERSION >= 25
+AGameModeBase* USpatialGameInstance::CreateGameModeForURL(FURL InURL, UWorld* InWorld)
```

Comment:
Although this approach is nice, I think we can just make a one line engine change similar to how we set bReplicates to true when running spatial - https://github.com/improbableio/UnrealEngine/blob/12dfe67f9f1bee648e3515b8577892bc0c29f95b/Engine/Source/Runtime/Engine/Private/GameModeBase.cpp#L68

Reply:
This is correct - we should probably discuss this with @ImprobableNic since he's endorsed the GameInstance override approach.